mne/io/nsx/nsx.py

```diff
@@ -0,0 +1,601 @@
+# Author: Porloy Das <pdas6@mgh.harvard.edu>
+#
+# License: BSD-3-Clause
+import os
+from datetime import datetime, timezone
+from warnings import warn
+
+import numpy as np
+
+from ..constants import FIFF
+from ..meas_info import _empty_info
+from ..base import BaseRaw
+from ..utils import _read_segments_file
+
+from ...annotations import Annotations
+from ...utils import logger, fill_doc
+
+
+# common channel type names mapped to internal ch types
+CH_TYPE_MAPPING = {
+    "EEG": FIFF.FIFFV_EEG_CH,
+    "SEEG": FIFF.FIFFV_SEEG_CH,
+    "CC": FIFF.FIFFV_SEEG_CH,
+    "ECOG": FIFF.FIFFV_ECOG_CH,
+    "DBS": FIFF.FIFFV_DBS_CH,
+    "EOG": FIFF.FIFFV_EOG_CH,
+    "ECG": FIFF.FIFFV_ECG_CH,
+    "EMG": FIFF.FIFFV_EMG_CH,
+    "BIO": FIFF.FIFFV_BIO_CH,
+    "RESP": FIFF.FIFFV_RESP_CH,
+    "TEMP": FIFF.FIFFV_TEMPERATURE_CH,
+    "MISC": FIFF.FIFFV_MISC_CH,
+    "SAO2": FIFF.FIFFV_BIO_CH,
+}
+
+DATA_BYTE_SIZE = 2
+ORIG_FORMAT = "short"
+
+_unit_range_dict = {"V": 1, "µV": 1e-6, "uV": 1e-6, "mV": 1e-3}  # V stands for Volt
```

Comment:
`from mne.io.base import _get_scaling` might get you (the inverse of) this already:
```
>>> _get_scaling('eeg', 'V')
1.0
>>> _get_scaling('eeg', 'uV')
1000000.0
>>> _get_scaling('eeg', 'µV')
1000000.0
>>> _get_scaling('eeg', 'mV')
1000.0
```